sale_coupon_limit/models/sale_coupon_rule.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleCouponRule(models.Model):
+    _inherit = "sale.coupon.rule"
+
+    rule_max_customer_application = fields.Integer(
+        string="Maximum Customer Applications",
+        default=0,
+        help="Maximum times that a program can be applied to a customer. "
+        "0 for no limit.",
+    )
+    rule_salesmen_limit_ids = fields.One2many(
+        string="Salesmen Limits",
+        comodel_name="sale.coupon.rule.salesmen.limit",
+        inverse_name="rule_id",
+        help="Maximum times a salesman can apply a program. 0 for no limit.",
+    )
+    rule_salesmen_restrict_list_only = fields.Boolean(
+        string="Restrict on listed salesmen",
+        default=True,
+        help="Check it in order to apply restrictions only to the salesmen configured "
+        "in the list.",
+    )
```

Comment:
This misses a compute field indicating the maximum number of salesmen number that sums up all the lines and being displayed on UI.

Reply:
I don't understand this. What's the purpose of that field?

Reply:
To get the total limit. It's a requirement of the customer.

Reply:
Ok, take a look now